staging/src/k8s.io/legacy-cloud-providers/aws/aws_routes.go

```diff
@@ -119,7 +119,7 @@ func (c *Cloud) ListRoutes(ctx context.Context, clusterName string) ([]*cloudpro
 				route.TargetNode = mapInstanceToNodeName(instance)
 				routes = append(routes, route)
 			} else {
-				klog.Warningf("unable to find instance ID %s in the list of instances being routed to", instanceID)
+				klog.InfoS("unable to find instance in the list of instances being routed to","routetable", r)
```

Comment:
Hi, this is a tiny suggestion:

```suggestion
				klog.InfoS("unable to find instance in the list of instances being routed to", "routetable", r)
```